packages/Webkul/Admin/src/Resources/lang/es/app.php

```diff
@@ -1495,8 +1495,8 @@
 
             'campaigns' => [
                 'index' => [
-                    'create-btn' => 'Create Campaign',
-                    'title'      => 'Campaigns',
+                    'create-btn' => 'Crear Campaña',
+                    'title'      => 'Campañas',
 
                     'datagrid' => [
                         'active'   => 'Active',
```

Comment:
Hello @dupotey ,
Can you please add this datagrid key translation as well? 

The datagrid key is not translated into Spanish.